bazel/jsonschema/BUILD

```diff
@@ -0,0 +1,334 @@
+load("@com_github_chrusty_protoc_gen_jsonschema//:defs.bzl", "jsonschema_compile")
+
+licenses(["notice"])  # Apache 2
+
+# generated manually using: bazel query 'filter("@envoy_api:*", kind(proto_library, deps(@envoy_api//:all_protos) except @envoy_api//:*))'
+# may be generated by tools/proto_format/proto_sync.py (like proto_library's deps in api/BUILD)?
+PROTOS = [
+    "@envoy_api//contrib/envoy/extensions/filters/http/dynamo/v3:pkg",
+    "@envoy_api//contrib/envoy/extensions/filters/http/golang/v3alpha:pkg",
```

Comment:
we dont want to do this

its possible that this list could be derived by using a `genquery` but almost certainly this should be implemented using an aspect, which allows you to traverse dependencies, and visit individual targets